Tasks/VsTest/Helpers.ps1

```diff
@@ -92,3 +96,33 @@ function SetupRunSettingsFileForParallel {
     return $runSettingsFilePath
 }
 
+function Get-SubKeysInFloatFormat($keys)
+{
+	$targetKeys = @()      # New array
+	foreach ($key in $keys)
+	{
+		$targetKeys += $key -as [decimal]
+	}
+
+	return $targetKeys
+}
+
+function Locate-VSVersion()
+{
+	#Find the latest version
+	$regPath = "HKLM:\SOFTWARE\Microsoft\VisualStudio"
+	if (-not (Test-Path $regPath))
+	{
+		return $null
+	}
+	
+	$keys = Get-Item $regPath | %{$_.GetSubKeyNames()} -ErrorAction SilentlyContinue
```

Comment:
Yes. Already encountered this. We are using -as [decimal] function, so that if it's not convertible type, it will ignore them